### ISSUE TITLE
Add autocomplete for "unset" command

### ISSUE THREAD

**Issue by jgtvares** [withfig/autocomplete#1811]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my feature

### Description:
The `echo` command shows available variables in the environment, but the `unset` command does not.
I work with multiple AWS profiles, some of them are just the usual credentials exports and others are with the `AWS_PROFILE` variable.
When I need to use `AWS_PROFILE`, I have to `unset` all other exported AWS variables because they have precedence over `AWS_PROFILE`.
I could just have a simple script for it, but if Fig had an autocomplete for `unset` I would be satisfied with it and ditch my script 😅 !
So, to sum up, when typing `unset`, Fig should present to me _unsetable_ variables.

**Comment by DarrenSeng** [comment]
Hey @jgtvares! This is fixed in #1858

**Comment by abhinav2712** [comment]
is this opened?
